docs/user/resources/04-80-azure-redis-instance.md

```diff
@@ -0,0 +1,72 @@
+| **shardCount**                                          | int    | Optional. Number of shards.                                                                                                                                                                                                             |
+| **replicasPerPrimary**                                  | int    | Optional. Number of replicas, for `Premium` tier it is [up to 3.](https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-high-availability)                                                                                |
+| **redisConfiguration**                                  | object | Optional.                                                                                                                                                                                                                               |
+| **redisConfiguration.maxclients**                       | int    | Optional. Max number of Redis clients. Limited to [7,500 to 40,000.](https://azure.microsoft.com/en-us/pricing/details/cache/)                                                                                                          |   
+| **redisConfiguration.maxmemory-reserved**               | int    | Optional. [Configure your maxmemory-reserved setting to improve system responsiveness.](https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-best-practices-memory-management#configure-your-maxmemory-reserved-setting) |
+| **redisConfiguration.maxmemory-delta**                  | int    | Optional. Gets or sets value in megabytes reserved for non-cache usage per shard e.g. failover.                                                                                                                                         | 
+| **redisConfiguration.maxmemory-policy**                 | int    | Optional. The setting for how Redis will select what to remove when maxmemory (the size of the cache offering you selected when you created the cache) is reached. Defaults to `volatile-lru`.                                          | 
+| **redisConfiguration.maxfragmentationmemory-reserved**  | int    | Optional. [Configure your maxmemory-reserved setting to improve system responsiveness.](https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-best-practices-memory-management#configure-your-maxmemory-reserved-setting) |
+
+# Auth Secret Details
+
+The following table list the meaningful parameters of the auth Secret:
+
+| Parameter                 | Type   | Description                                                                                            |
+|---------------------------|--------|--------------------------------------------------------------------------------------------------------|
+| **.metadata.name**        | string | Name of the auth Secret. It will share the name with the AzureRedisInstance unless specified otherwise |
```

Comment:
```suggestion
| **.metadata.name**        | string | Name of the auth Secret. It shares the name with AzureRedisInstance unless specified otherwise |
```

Reply:
Fixed.